index.bs

```diff
@@ -7149,7 +7149,7 @@ script.RemoteValue = (
   script.WindowProxyRemoteValue
 )
 
-script.InternalId = js-uint;
+script.InternalId = text;
```

Comment:
One question that I have should this definition have a separate block in Types similar to other IDs (https://w3c.github.io/webdriver-bidi/#type-script-SharedId or https://w3c.github.io/webdriver-bidi/#type-browsingContext-Browsingcontext) ?

Reply:
This might probably be a good idea.

Reply:
Done.